data/mods/Magiclysm/martialarts.json

```diff
@@ -15,7 +15,8 @@
         "stonefist",
         "stormglove",
         "rune_earthshaper_weapon",
-        "shotcestus"
+        "shotcestus",
+        "rune_earthshaper_weapon"
```

Comment:
looks like a duplicate

Reply:
Dunno bud, they only appear once for me so, I can't really delete something that isn't there.